src/app/backend/deploy.go

```diff
@@ -243,8 +243,20 @@ func getLabelsMap(labels []Label) map[string]string {
 	return result
 }
 
+type ObjectFromInfo interface {
+	createObjectFromInfo(info *kubectlResource.Info) error
+}
+
+type ResourceFromInfo string
```

Comment:
You dont need to do this. I.e., creating fake struct that fulfills an interface. You can just have a callback function as the argument. @floreks did this recently in PR https://github.com/kubernetes/dashboard/pull/319 , you can talk with him.